dune-project

```diff
@@ -115,6 +115,7 @@
     (alcotest :with-test)
     base-unix
     fmt
+    (bisect_ppx :build)
```

Comment:
should it also have a :with-test as well?

Reply:
I'll have to try whether you can build without (it is off by default, but present in the dune as instrument declaration). It'd certainly be desirable to limit where it is needed

Reply:
Uninstalled `bisect_ppx` and building+runtest still worked (`make coverage` obviously didn't), so I change this to ':with-test'.
This is different to how bisect-ppx worked in the past where it was always required to be present at build time even when unused, so this is a nice upstream improvement.